test/bats/helpers/transactions.bash

```diff
@@ -0,0 +1,16 @@
+count_transactions_by_currency() {
+  transactions_array_query=$1
+  currency=$2
+
+  echo $output \
+    | jq -r "$transactions_array_query" \
+    | jq -r --arg currency "$currency" 'select(.node.settlementCurrency == $currency)' \
```

Comment:
does not work just with ` | length`

Reply:
I did it like this for easier readability. Can go either way if we really want to change it.